DemoApp/DemoApp/DemoApp.csproj

```diff
@@ -112,6 +112,9 @@
 	  <MauiXaml Update="Views\TMRadioButtonPage.xaml">
```

Comment:
Not working in iOS. OnPaintSurface never gets called when running in iOS.

![Simulator Screen Shot - iPhone 14 - 2023-06-27 at 15 41 28](https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/33b1e43f-8aba-4f6b-aa23-edf50dd3a5a3)

Reply:
I changed the linking behaviour and it worked.